proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -967,11 +994,13 @@ void getChar() throws IOException {
     prevLine = currLine;
     prevCol = currCol;
     currFile = prepro.getFileIndex();
-    currLine = prepro.getLine();
+    currLine = prepro.getLine2();
     currCol = prepro.getColumn();
   }
 
   void macroDefine(int defType) throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)